magento/magento2
# Fix bug 13314: Magento 2 Tier Pricing Not Working As Expected on Product Page.

### Description (*)
Fix bug 13314: Magento 2 Tier Pricing Not Working As Expected on Product Page.

### Related Pull Requests


### Fixed Issues (if relevant)

1. magento/magento2#13314: Magento 2 Tier Pricing Not Working As Expected on Product Page.

### Manual testing scenarios (*)

1. I have added a set number of tiered pricing for product "A" which it's base price is £50 and set 5 tiers as below:

 - Buy 5 at £45 (All Groups "General, Guest & wholesale)
 - Buy 10 at £40 (All Groups)
 - Buy 15 at £35 (All Groups)
 - Buy 20 at £30 (All Groups)
 - Buy 25 at £25 (All Groups)

I have set a specific wholesale price tier for:

 - Buy 15 at £20 (Wholesale Only)

But on my frontend the tier pricing for a wholesale customer looks like this:

 - Buy 5 at £45 (All Groups) <---- This is working as expected.
 - Buy 10 at £40 (All Groups) <---- This is working as expected.
 - Buy 15 at £20 (Wholesale) <---- This is working as expected.
 - Buy 20 at £30 (All Groups) <---- Should be hidden.
 -  Buy 25 at £25 (All Groups) <---- Should be hidden.

### Questions or comments


### Contribution checklist (*)
